connector/protobuf/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a Jira open for this feature. We can open an EPIC. I am thinking of adding tasks under it. 
cc: @mposdev21

Reply:
No, there is no JIRA. Please go ahead and create the. EPIC.

Reply:
Created an Epic with 5 tasks under it: https://issues.apache.org/jira/browse/SPARK-40653 
I see that PR title is already updated :). Nice.